auth/src/include/firebase/auth.h

```diff
@@ -174,7 +174,7 @@ class Auth {
   void set_language_code(const char* language_code);
 
   /// Sets the user-facing language code to be the default app language. This
-  /// uses a languge associated with the phone's locale data.  On desktop
+  /// uses a languge associated with the device's locale data.  On desktop
```

Comment:
While you're here, can you fix the typo "languge" => "language"?

Reply:
:+1: